engines/group/app/controllers/group/api/v1/memberships_controller.rb

```diff
@@ -9,7 +9,8 @@ class MembershipsController < ApplicationController
         # GET /api/v1/memberships
         #
         def index
-          memberships = Membership.where(user_id: current_user.id)
+          group_ids = current_user.groups.map(&:id)
+          memberships = Membership.where(group_id: group_ids)
```

Comment:
Just one thing, let's make sure we move this logic to the model layer as soon as we need it again somewhere else.